modules/custom/ginvite/config/update/ginvite_update_11401.yml

```diff
@@ -0,0 +1,20 @@
+views.view.group_invitations:
```

Comment:
We changed ginvite from custom to community module, I'm not sure we should modify the community module. Can we remove this settings?

Reply:
@tbsiqueira done